catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -110,7 +113,7 @@ public MetacardImpl(MetacardType type) {
          * serialized object is maintained. For instance, if a null check is added in the
          * constructor, the same check should be added in the readObject() method.
          */
-        map = new HashMap<String, Attribute>();
+        map = new ConcurrentHashMap<>();
```

Comment:
Just to be sure, Can we not guard the place that is concurrently accessing this over using a concurrent hashmap? These internals are *really* heavily accessed, so this could bring performance implications.